src/background.js

```diff
@@ -1,6 +1,7 @@
-chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+const browser = ((a, b) => a && a.runtime ? a : b)(this.chrome, this.browser);
```

Comment:
Can you please rewrite this as something like this:
```javascript
const getBrowser = () => {
  if (this.chrome && this.chrome.runtime) {
    return this.chrome;
  }
  return this.browser;
}
```

I know it's way more verbose with 6 lines instead of 1, but it's easier for me personally to understand.

Reply:
I would normally favour a longer form like your posted example, but this particular line has to exist in both background.js and extension.js.

How do you feel about:

```js
const browser = this.chrome && chrome.runtime ? chrome : this.browser;
```

Once the TypeScript migration is complete it can be moved to a module (`import browser from './browser'`).

Reply:
LGTM, please use that line instead